extensions/localPublish/package.json

```diff
@@ -13,20 +13,25 @@
     "@botframework-composer/types": "file:../../Composer/packages/types",
     "adm-zip": "^0.4.14",
     "archiver": "^5.0.2",
+    "express": "^4.17.1",
+    "express-serve-static-core": "^0.1.1",
```

Comment:
is this `...-serve-static-core` package needed? I'm not seeing it used anywhere in the diff

Reply:
Nice catch. I had ported it from server but things work fine without it.